api/server.go

```diff
@@ -760,14 +781,16 @@ func (s *server) exchangeRateHandler(jc jape.Context) {
 }
 
 // NewServer returns an HTTP handler that serves the explored API.
-func NewServer(e Explorer, cm ChainManager, s Syncer, ex exchangerates.Source) http.Handler {
+func NewServer(e Explorer, cm ChainManager, s Syncer, ex exchangerates.Source, manualScanPassword string) http.Handler {
 	srv := server{
-		cm:        cm,
-		e:         e,
-		s:         s,
-		ex:        ex,
-		startTime: time.Now().UTC(),
+		cm:                 cm,
+		e:                  e,
+		s:                  s,
+		ex:                 ex,
+		manualScanPassword: manualScanPassword,
```

Comment:
I'd just call this `apiPassword`. We probably also want to protect the endpoint for connecting to peers now that I'm looking at the routes. Because letting someone else decide what peers you are connected to isn't very secure.